cmd/orch-tester/orch_tester.go

```diff
@@ -80,14 +118,15 @@ func main() {
 	defer cancel()
 
 	if *streamTester == "" {
-		glog.Info("Starting embedded streamtester service")
-		hostName, _ := os.Hostname()
-		streamtesterMetrics.InitCensus(hostName, model.Version, "streamtester")
-		s := server.NewStreamerServer(false, streamTesterLapiToken, streamTesterMistCreds, 4242)
-		go func() {
-			addr := fmt.Sprintf("%s:%s", "0.0.0.0", streamTesterPort)
-			s.StartWebServer(ctx, addr)
-		}()
+		startEmbeddedStreamTester(ctx)
+	}
+
+	var bcastHost string
+	if *broadcaster == "" {
```

Comment:
Is it possible to use only the embedded stream-tester or broadcaster but not both? It seems like at least the embedded broadcaster is configured explicitly with the local stream tester (e.g. using `defaultHost` for the orch webhook). I think that's completely fine, but maybe we should have a check here to make sure that either none or both of these fields are set, failing the `orch_tester` with an error otherwise.

Reply:
Good point. updated in ee607027a515b64263b6972cf2eb839f1df9563f